### ISSUE TITLE
Share button when only HTML / CSS is coded

### ISSUE THREAD

**Issue by DeepaMuralidhar** [PencilCode/pencilcode#204]
Under setting have HTML / CSS checked. When no programs have been written in the programming space, but the HTML space there is content, click play to see the web page. Now click Share to copy-paste code.
The button does not work.

**Comment by AmayAgrawal** [comment]
I don't think so that the code should be shared when there is empty programming space.If, then also requested to do so we have to just remove one check.We can do it if you think so

**Comment by davidbau** [comment]
I do think it should be possible to share programs with just HTML and no
JS/CS.  In fact, I wonder if we should have another choice for JS/CS
language which is "None", so you can have a an entire program which is just
a static HTML page.

**Comment by AmayAgrawal** [comment]
No, at present we are not able to share anything if the js/cs is empty.

**Comment by davidbau** [comment]
Understood.  This is a bug and it should be fixed.
